invenio.cfg

```diff
@@ -592,8 +592,8 @@ _RECORD_EXPORTERS = {
         "filename": "{id}-allversions.bib",
     },
 }
-APP_RDM_RECORD_EXPORTERS = dict(sorted(_RECORD_EXPORTERS.items()))
 
 ADMINISTRATION_BASE_TEMPLATE = "cds_rdm/administration/admin_base_template.html"
 LOGGING_CONSOLE_LEVEL = "INFO"
 JOBS_LOGGING_LEVEL = "INFO"
+RDM_ALLOW_OWNERS_REMOVE_COMMUNITY = False
```

Comment:
for reference: ongoing naming discussion https://github.com/inveniosoftware/invenio-rdm-records/pull/2218